cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -72,6 +72,7 @@ endif ()
 # ----------------------------------------------------------------------
 # ORC Format
 ExternalProject_Add (orc-format_ep
+  URL "https://dlcdn.apache.org/orc/orc-format-${ORC_FORMAT_VERSION}/orc-format-${ORC_FORMAT_VERSION}.tar.gz"
   URL "https://archive.apache.org/dist/orc/orc-format-${ORC_FORMAT_VERSION}/orc-format-${ORC_FORMAT_VERSION}.tar.gz"
```

Comment:
According to https://cmake.org/cmake/help/latest/module/ExternalProject.html#url , this looks wrong to me. Have you tested this PR, @progval ?
```
URL <url1> [<url2>...]
```

Reply:
the behavior is the same either way. I checked by replacing the first URL with a non-existing one.